src/main/resources/templates/fragments/head.html

```diff
@@ -20,8 +20,12 @@
 
     <link href="../css/sidemenu.css" rel="stylesheet" />
 
+   
+
     <link rel="stylesheet" href="../js/codemirror/lib/codemirror.css" />
 
+    <link rel="stylesheet" href="../js/Monaco-Editor/dev/vs/editor/editor.main.css" />
+
```

Comment:
i see you added the worker for file code from codemirror
I think you need to include it here and also the `basic-language/python` and `basic-language/bash`